scrapinghub/hubstorage/collectionsrt.py

```diff
@@ -32,7 +32,7 @@ def _allows_mpack(self, path=None):
         :param path: None, tuple or string
 
         """
-        if not MSGPACK_AVAILABLE:
+        if not MSGPACK_AVAILABLE or not self.client.use_msgpack:
```

Comment:
Looks better, thanks. Btw in both cases if messagepack is not available - it will silently ignore `use_msgpack` option, does it make sense to add some warnings for such case?